app/views/courses/_course_table_row.html.erb

```diff
@@ -1,41 +1,30 @@
 <tr class="govuk-table__row">
   <td class="govuk-table__cell course-table__course-name" data-qa="courses-table__course">
-    <%= govuk_link_to "#{ course.attributes[:name] } (#{ course.attributes[:course_code] })",
-      manage_ui_course_page_url(provider_code: @provider[:provider_code], course_code: course.attributes[:course_code]),
+    <%= govuk_link_to "#{course.name_and_code}",
+      manage_ui_course_page_url(provider_code: @provider.provider_code, course_code: course.course_code),
       class: "govuk-link govuk-heading-s govuk-!-margin-bottom-0"
     %>
-    <span class="govuk-body-s"><%= course.attributes[:description] %></span>
+    <span class="govuk-body-s"><%= course.description %></span>
   </td>
   <td class="govuk-table__cell" data-qa="courses-table__ucas-status">
-    <%= course_ucas_status(course) %>
+    <%= course.ucas_status %>
   </td>
   <td class="govuk-table__cell" data-qa="courses-table__content-status">
-    <%= render partial: 'content_status_tag', locals: { course: course } unless course.not_running? %>
+    <% unless course.not_running? %>
```

Comment:
Can we do this check in the presenter and return nil if it's not running?

Reply:
Done